qiskit/algorithms/amplitude_estimators/ae.py

```diff
@@ -458,7 +458,6 @@ def compute_confidence_interval(
             The (1 - alpha) confidence interval of the specified kind.
 
         Raises:
-            AquaError: If 'mle' is not in self._ret.keys() (i.e. `run` was not called yet).
```

Comment:
Obviously `AquaError` is wrong here, but as best as I could tell it was already out-of-date when it was merged in #5517, which seems to have involved some refactoring from old Aqua code that I couldn't be bothered to track down.

Reply:
:) I happened to notice this too in qiskit-algorithms as that was prepared for release and it was taken care of there. I figured it had been here for years like this since it was brought over from Aqua and never noticed, and as it was not causing an issue and with algorithms set to be removed I.... (was the same "fix" we did there!)

Reply:
This is from an old (very old) version of the code, where the algorithm classes where still stateful. Back then you had to run the algorithm to generate some info and then this enabled some post processing options.